build-tools/upload-junit-to-cloud.py

```diff
@@ -1,30 +1,52 @@
+import typing
+
 from google.cloud import bigquery
 import xml.etree.ElementTree as ET
 import argparse
+from typing import List, Optional
+import glob
+
 
 # Uploading JUnit test results to BigQuery
-def upload(final: bool):
+def upload(final: bool, dummy: bool, url: Optional[str], files: List[typing.TextIO]):
```

Comment:
info: we need to support multiple files upload to accommodate results of unit tests and integration tests.